### ISSUE TITLE
Active series in the tooltip is not highlighted enough in 8.0.0

### ISSUE THREAD

**Issue by lilydjwg** [grafana/grafana#35397]
**What happened**:
I upgraded from 7.5.7 to 8.0.0, and the tooltip no longer highlights the series name the mouse cursor points to.

One old instance that hasn't been upgraded still works like before so this should not be that my browser has been upgraded.

**What you expected to happen**:
Nothing broken.

**How to reproduce it (as minimally and precisely as possible)**:
Just add more than one series into a graph and set "Tooltip mode" to "All". Move the mouse cursor to the data points.

**Environment**:
- Grafana version: 8.0.0
- Data source type & version: graphite 1.1.3
- OS Grafana is installed on: Arch Linux
- User OS & Browser: Arch Linux, Firefox 91.0a1
- Grafana plugins: none.
- Others:

**Comment by torkelo** [comment]
Strange, unable to replicate this. I see it in bold, it's not as noticeable as we fixed the so it using the correct weight (semi bold), so we should find something else to use to highlight it

**Comment by lilydjwg** [comment]
OK, I find it: it sets `font-weight: 500`, and my font in use is Helvetica, but it doesn't have that weight and fallbacks to `Regular`. `500` is not `semibold` (which fallbacks to `bold`); [it's `Medium`](https://developer.mozilla.org/en-US/docs/Web/CSS/font-weight#common_weight_name_mapping).

When I disable all `font-family` rules on the page, it's showing bolder using my default font `DejaVu Sans`. I don't know why people are picking particular fonts for the page disregard the user's choice, especially old fonts like Helvetica or the like.... My DejaVu Sans font seems to support both 200 and 500 weights and Helvetica supports none of them.

**Comment by lilydjwg** [comment]
I made a workaround by adding a stylus style to grafana and the tooltip looks much nicer:

```css
body {
	font-family: sans !important;
}
```

**Comment by torkelo** [comment]
have you overridden the default font?

**Comment by lilydjwg** [comment]
There is no default font on Arch Linux. Users just install whatever they want.

I've uninstalled Helvetica since it seems to only cause troubles for me. But now it fallbacks to Nimbus Sans (provided by the `gsfonts` package) which fontconfig thinks it's the most compatible font available, and so it has the same problem as Helvetica.

There do exist differences on Windows 10 (without installing any additional fonts) but it's less noticeable.

**Comment by tfylling** [comment]
On Windows I do see that there is a difference, in both Vivaldi, Chrome and Firefox, but it's so slight that I didn't know there was one until I came here report it and read this. Even now I have to move the mouse cursor between the different series multiple times to notice a very few subtle pixel shifts. Couldn't it also get a slightly brighter color to make it stand out a bit more?
![image](https://user-images.githubusercontent.com/20243684/123040744-87dfbc80-d3f4-11eb-8bc4-119ca289a302.png)

**Comment by dprokop** [comment]
Would something like this work - series icon slightly bolder when series is active + shifted to the left to make it stick out. WDYT @torkelo?

<img width="788" alt="Screen Shot 2021-07-13 at 14 55 09" src="https://user-images.githubusercontent.com/2376619/125455172-8a13c096-0622-406a-9e74-045a870a953c.png">
